tests/unit/test_commands.py

```diff
@@ -26,16 +26,14 @@
         [],
         ["-h"],
         ["--help"],
-        ["help"],
```

Comment:
Why are we removing this test?

Reply:
This was a mistake, thanks for pointing it out. Will add it back in a new patch